### DISCUSSION TITLE
Remote cache not working `turbo run ... --token=xxx`

### DISCUSSION THREAD

**Discussion by JustGreg** [vercel/turborepo#714] [Help] [answered]
When i use `turbo run ... --token=xxx -vvv` turbo is not fetching https://api.vercel.com/v8/artifacts/xyz.
It only works with `--token=xxx --team=my-team` but this will fetch https://api.vercel.com/v8/artifacts/xyz?slug=my-team

Is there a way to fetch https://api.vercel.com/v8/artifacts/xyz without team/slug? (this should run in CI/CD therefore i can't do turbo login and turbo link)

**Comment by anthonyshew** [comment, marked as answer]
Environment variables can be set for these values in CI. Some examples can be found here: https://turbo.build/repo/docs/ci
